src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -157,8 +157,25 @@ namespace AppInstaller::CLI::Workflow
         auto packages = PackagesJson::CreateJson(context.Get<Execution::Data::PackageCollection>());
 
         std::filesystem::path outputFilePath{ context.Args.GetArg(Execution::Args::Type::OutputFile) };
+
+        // Check if the file exists and is hidden
+        DWORD attrs = std::filesystem::exists(outputFilePath) ? GetFileAttributesW(outputFilePath.c_str()) : INVALID_FILE_ATTRIBUTES;
+        bool isHidden = (attrs != INVALID_FILE_ATTRIBUTES && (attrs & FILE_ATTRIBUTE_HIDDEN));
+
+        if (isHidden)
+        {
+            // Remove hidden attribute so we can write to it
+            SetFileAttributesW(outputFilePath.c_str(), attrs & ~FILE_ATTRIBUTE_HIDDEN);

```

Comment:
I would really like it if there were a way to simply open the hidden file rather than changing the attribute.  There might be a set of mode options that change the `CreateFile` call to avoid the documented error state (I would think specifically anything that moves to "open only existing files" might do it.  So more of a `if (exists) open_existing_with_truncate() else create()`.

If that cannot be done, then at a minimum we should use a scope exit to reset the attributes.